src/component/main/main/Main.tsx

```diff
@@ -1,37 +1,67 @@
-import { useNavigate } from 'react-router-dom';
-import { useContext } from 'react';
+import { useContext, useState } from 'react';
 import Map from './Map';
 import LostList from './list/LostList';
-import { SelectedModeContext } from '../../../contexts/AppContexts';
-import { SelectedAreaIdContext } from '../../../contexts/AppContexts';
+import { SelectedModeContext, TotalCountContext } from '../../../contexts/AppContexts';
+import { useNavigate } from 'react-router-dom';
 
 const Main = () => {
-  const navigate = useNavigate();
   const { selectedMode, setSelectedMode } = useContext(SelectedModeContext)!;
-  const { selectedAreaId } = useContext(SelectedAreaIdContext)!;
+  const [isMobileListOpen, setIsMobileListOpen] = useState(false);
+  const { totalCount } = useContext(TotalCountContext)!;
+
+  const navigate = useNavigate();
 
   const handleRegisterButtonClick = () => {
-    if (selectedMode === 'register') {
-      setSelectedMode('append');
-    } else {
-      navigate(`/register/${selectedAreaId}`);
-    }
+    setSelectedMode(selectedMode === 'register' ? 'append' : 'register');
+    const url = new URLSearchParams();
+    url.set('schoolAreaId', '0');
+    url.set('page', '1');
+    url.set('categoryId', '0');
+    navigate(`${url.toString()}`);
```

Comment:
https://github.com/user-attachments/assets/9ae68e0e-f5bc-4ecc-b5a9-93139d2ad7c9


위 영상처럼 등록 버튼과 조회버튼을 클릭할 때, 파라미터가 계속해서 누적되는 현상이 발생합니다. 아래의 코드로 수정하여 파라미터 누적 현상을 해결하면 좋을 것 같습니다!
```suggestion
    navigate({ search: `?${url.toString()}` }, { replace: true });
```

Reply:
수정했습니다!